### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by wlthomas**
Documentation update based on review by Adrian Dusa

### CODE REVIEW

File: schemas/codebook.xsd [hunk 1]
```diff
@@ -1924,6 +1924,12 @@ for more details.
             <xs:attribute name="vocab" type="xs:string"/>
             <xs:attribute name="vocabURI" type="xs:string"/>
             <xs:attribute name="vocabInstanceURI" type="xs:string"/>
+			<xs:attribute name="vocabID" type="xs:string"/>
+			<xs:attribute name="vocabAgencyName" type="xs:string"/>
+			<xs:attribute name="vocabVersionID" type="xs:string"/>
+			<xs:attribute name="otherValue" type="xs:string"/>
+			<xs:attribute name="vocabSchemeURN" type="xs:string"/>
+			<xs:attribute name="vocabInstanceCodeTerm" type="xs:string"/>
```
Comment (DanSmith) [inline schemas/codebook.xsd:1932]: non-documentation change

File: schemas/codebook.xsd [hunk 2]
```diff
@@ -2422,7 +2428,7 @@ for more details.
       <xs:complexContent>
          <xs:extension base="baseElementType">
             <xs:sequence>
-	           <xs:element name="typeOfAccess" type="conceptType" minOccurs="0"/>
+	           <xs:element name="typeOfAccess" type="conceptType" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (DanSmith) [inline schemas/codebook.xsd:2431]: non-documentation change

File: schemas/codebook.xsd [hunk 3]
```diff
@@ -5160,7 +5172,7 @@ for more details.
       <xs:complexContent>
          <xs:extension base="baseElementType">
             <xs:sequence>
-	       <xs:element name="typeOfAccess" type="conceptType" minOccurs="0"/>
+	           <xs:element name="typeOfAccess" type="conceptType" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (DanSmith) [inline schemas/codebook.xsd:5175]: non-documentation change

File: schemas/codebook.xsd [hunk 4]
```diff
@@ -5515,6 +5527,12 @@ for more details.
             <xs:attribute name="vocab" type="xs:string"/>
             <xs:attribute name="vocabURI" type="xs:string"/>
             <xs:attribute name="vocabInstanceURI" type="xs:string"/>
+			<xs:attribute name="vocabID" type="xs:string"/>
+			<xs:attribute name="vocabAgencyName" type="xs:string"/>
+			<xs:attribute name="vocabVersionID" type="xs:string"/>
+			<xs:attribute name="otherValue" type="xs:string"/>
+			<xs:attribute name="vocabSchemeURN" type="xs:string"/>
+			<xs:attribute name="vocabInstanceCodeTerm" type="xs:string"/>
```
Comment (DanSmith) [inline schemas/codebook.xsd:5535]: non-documentation change

File: schemas/codebook.xsd [hunk 5]
```diff
@@ -7111,7 +7131,7 @@ for more details.
          <xs:extension base="baseElementType">
             <xs:choice>
                <xs:sequence>
-                  <xs:element name="typeOfDataSrc" type="conceptType" minOccurs="0"/>
+                  <xs:element name="typeOfDataSrc" type="conceptType" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (DanSmith) [inline schemas/codebook.xsd:7134]: non-documentation change

File: schemas/codebook.xsd [hunk 6]
```diff
@@ -7499,7 +7519,7 @@ for more details.
       <xs:complexContent>
          <xs:extension base="baseElementType">
             <xs:sequence>
-               <xs:element name="typeOfDataSrc" type="conceptType" minOccurs="0"/>
+               <xs:element name="typeOfDataSrc" type="conceptType" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (DanSmith) [inline schemas/codebook.xsd:7522]: non-documentation change

**Comment by wlthomas** [comment]
Also includes added attributes to concept (CV matchup with Lifecycle and added item code) plus relaxing cardinality to support multiple language entries.

**Pull request merged**